### DISCUSSION TITLE
a problem about enterGroup

### DISCUSSION THREAD

**Discussion by wuxingping** [fabricjs/fabric.js#8764] [General]
if the group has been rotated,the object would jump slightly when it entered the group by the method of enterGroup as the following video.

https://user-images.githubusercontent.com/41353266/223349353-063c08ee-fd1b-4801-bcc6-29498ef6a70b.mp4

**Comment by wuxingping** [comment]
if the group hasn't been rotated,it works well.

https://user-images.githubusercontent.com/41353266/223353368-d596e610-c6ea-4232-b10d-a383402ca0ce.mp4

**Comment by ShaMan123** [comment]
This looks like a bug.
Can you create a reproduction?

**Reply by wuxingping** [reply]
> This looks like a bug. Can you create a reproduction?

you mean create a reproduction online?the logic is a little complicated,i am afraid i can't create a reproduction totally online.i think the problem lies in the method of enterGroup.the key of the problem is the group is rotated.i guess maybe there is a bug or not prefect on the method of enterGroup.i hope you give me some advices about that.

**Reply by ShaMan123** [reply]
yes. I can't do anything without a minimal (so no other stuff interfere) online app the recreates the issue.

**Reply by wuxingping** [reply]
> yes. I can't do anything without a minimal (so no other stuff interfere) online app the recreates the issue.

there is a reproduction.thanks for checking.
https://codesandbox.io/s/border-problem-of-group-forked-g4drpm?file=/src/App.vue
